packages/data-bite/src/CdcDataBite.tsx

```diff
@@ -229,22 +310,20 @@ const CdcDataBite = (
         dataBite = Math.max(...numericalData);
         break;
       case DATA_FUNCTION_MIN:
-        dataBite = Math.min(...numericalData);
+        dataBite =Math.min(...numericalData);
         break;
       case DATA_FUNCTION_MODE:
-        dataBite = getMode(numericalData).join(', ');
+        dataBite = getMode(numericalData).join(''); // Needs to be properly tested !!!
         break;
       case DATA_FUNCTION_RANGE:
-        numericalData.sort((a, b) => a - b);
-        let rangeMin = applyPrecision(numericalData[0]);
-        let rangeMax = applyPrecision(numericalData[numericalData.length - 1]);
-
-        if (config.dataFormat.commas) {
-          rangeMin = Number(rangeMin).toLocaleString('en-US');
-          rangeMax = Number(rangeMax).toLocaleString('en-US');
-        }
-
-        dataBite = config.dataFormat.prefix + applyPrecision(rangeMin) + config.dataFormat.suffix + ' - ' + config.dataFormat.prefix + applyPrecision(rangeMax) + config.dataFormat.suffix;
+        const sortedNumArr = [...numericalData].sort((a, b) => a - b);  // create shallow copy and sort
+        let rangeMin =  applyPrecision((sortedNumArr[0]))
```

Comment:
@Atash3000 - can you remove the sortedNumArr here and use Math.min and Math.max? If not the way you are using sortedNumArr looks fine.

Reply:
yes i ll remove.originally it was sorting there but sort affects original array and i copyied it. i think Math.max/min ll do better job.other thing is when we have data as string allways we get NaN or simple string concatinations, so there is either we convert all strings to number and then calculate like using reduce method or we ll just assume users ll work with string and number together.